### DISCUSSION TITLE
Allow menu folders to have a separate display title besides key name

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13311] [Feature Requests]
It's currently impossible to have two folders with the same name (group name needs to be unique) so it'd be nice to be able to specify the display name separately.

This can them remove the need for the keyname as that will be managed internally

**Comment by rijkvanzanten** [comment]
The display name can be configured under _Translations_ 👍🏻  The _Folder Key_ is the unique identifier of the folder, _Translations_ controls how the folder is rendered in the various languages the app supports.

**Reply by sxsws** [reply]
oh nice! I think that's initially unintuitive, but it makes sense in an 'ok' sort of way! maybe a translation for the base language can be added to the main form as the "display title" to help improve usability. Maybe this can go on the backburner and I can have a stab at it! I assume collections have a translation too that behaves the same?

**Reply by rijkvanzanten** [reply]
Yup! The same applies to collections / fields 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
